### PR TITLE
add test for stack mode

### PR DESCRIPTION
**Pull request by Kreeg**

### CODE REVIEW

**Comment by Kreeg** [comment]
I'm sure we can drop 2048x2048 svg I've added for calculations tests. It causes some glitches in png references (img is way bigger than it's parent. Or we can adjust css to deal with such images and update all expected references.

File: test/tmpl/stack.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                padding: 0;
+                text-align: center;
+                display: inline-block;
+                line-height: 1;
+                vertical-align: middle;
+            }
+            figcaption {
+                display: none;
+            }
+            i {
```
Comment (XhmikosR) [inline test/tmpl/stack.html:47]: The tmpl doesn't have any `i` elements. Should the test have them?

**Comment by XhmikosR** [comment]
> I'm sure we can drop 2048x2048 svg I've added for calculations tests. It causes some glitches in png references (img is way bigger than it's parent. Or we can adjust css to deal with such images and update all expected references.

As long as we still test things out, you can make a PR and we discuss it :)

File: test/tmpl/stack.html (hunk 1 above)
Comment (Kreeg) [reply test/tmpl/stack.html:47]: I think no. I'll remove it

**Pull request merged**